src/js/pages/Hukamnama/index.js

```diff
@@ -1,12 +1,12 @@
 /* globals API_URL */
 import React from 'react';
-import { buildApiUrl } from '@sttm/banidb';
+//import { buildApiUrl } from '@sttm/banidb';
 import PageLoader from '../PageLoader';
 import Layout, { Stub } from './Layout';
 
 export default class Hukamnama extends React.PureComponent {
   render() {
-    const url = buildApiUrl({ hukam: true, API_URL });
+    const url = `${API_URL}hukamnamas`;
```

Comment:
We can update banidb package.

Reply:
Bumped the version of banidb in my existing PR.

Reply:
We want to use buildApiUrl here right?